File: lab01/graficar.py
import pandas as pd
import matplotlib.pyplot as plt
import numpy as np

df = pd.read_csv('market-price.csv')

x = df.Timestamp.apply(lambda datetime: datetime.split()[0])
y = df['market-price']

plt.plot(x,y)
plt.xticks(np.linspace(0,365,8,dtype=int))
plt.tick_params(labelsize=13)
plt.grid()
plt.xlabel("Fecha", color="green", fontsize="17", fontweight="bold")
plt.ylabel("Precio en $", color="green", fontsize="17", fontweight="bold")
titulo = "Precio diario del Bitcoin durante el ultimo año."
plt.title(titulo, color="red", fontsize="24", fontweight="bold")
plt.show()

File: lab02/graficar.m
pkg load io
data = csv2cell('market-price.csv');

ycell = data(2:end,2);
y = cell2mat(ycell);

datetime = data(2:end,1);
mydate = cellfun(@(x) strsplit(x){1}, datetime,'uniformoutput',false);

xindex = round(linspace(1,366,8));

y2 = movmean(y,7);

plot(y2,'LineWidth',1);
set(gca, 'xtick', xindex,"fontsize", 15);
grid();
titulo = "Precio semanal de Bitcoin durante el ultimo año.";
title(titulo,'fontsize',25);
ylabel("Precio en $",'fontsize',20);
xlabel("Fecha",'fontsize',20);
xticklabels(mydate(xindex))
#yticklabels({'min','y = 0','max'})